core/controllers/admin.py

```diff
@@ -480,11 +480,11 @@ def _load_dummy_new_structures_data(self):
                     1, topic_id_1))
             # These explorations were chosen since they pass the validations
             # for published stories.
-            self._reload_exploration('15')
+            self._reload_exploration('6')
             self._reload_exploration('25')
             self._reload_exploration('13')
```

Comment:
**Note:** exploration 15 is failing the curated explorations check for "outcomes should not have training data". We also have a suite of unit tests for training data based on exploration 15, so I didn't want to change the YAML for it. So, I've switched it with exploration 6. Is this okay? Not sure who to tag -- @seanlip

Reply:
Yup that's fine, thanks!